aspnetcore/fundamentals/logging/index.md

```diff
@@ -125,6 +125,16 @@ In the preceding sample:
   * `AzureAppServicesBlob`
   * `ApplicationInsights`
 
+## Log in Program.cs
+
+The following example calls <xref:Microsoft.Extensions.Logging.ConsoleLoggerExtensions.AddConsole(ILoggerFactory)> in `Program.c` and logs the `/Test` endpoint:
```

Comment:
@guardrex  Line 130: [Warning] Cross reference not found: 'Microsoft.Extensions.Logging.ConsoleLoggerExtensions.AddConsole(ILoggerFactory)'.

Reply:
That tool calculates/guesses some of them. I think it should be ...

```
<xref:Microsoft.Extensions.Logging.ConsoleLoggerExtensions.AddConsole%2A>
```